medicaltorch/transforms.py

```diff
@@ -182,15 +200,16 @@ def __call__(self, sample):
         rdict = {}
         input_data = sample['input']
 
-        w, h = input_data.size
+        w, h = input_data[0].size
         th, tw = self.size
         fh = int(round((h - th) / 2.))
         fw = int(round((w - tw) / 2.))
 
         params = (fh, fw, w, h)
         self.propagate_params(sample, params)
```

Comment:
Not sure: if the input contrasts do not have the same size, then these params are not the same across contrasts --> is it robust to this aspect?

Reply:
Nop, you're right! What about [this](https://github.com/neuropoly/medicaltorch/blob/ab/generalization_dataset/medicaltorch/transforms.py#L215-L220)? Do we need to save the parameters for every contrast in the Ground Truth's metadata?

Reply:
Sorry for the late reply .. ..
Looks good
"for every contrast in the Ground Truth's metadata" --> I would say: in GT_metadata --> params of gt_crop while input1_metadata --> params of input1_crop.